src/ontology/cl-edit.owl

```diff
@@ -4941,13 +4942,11 @@ SubClassOf(obo:CL_0000149 obo:CL_0000147)
 
 # Class: obo:CL_0000150 (glandular epithelial cell)
```

Comment:
I am finding multiple daughter cells that do not fit with the new definition, or at least they are ambiguous:
- [colon glandular cell](http://purl.obolibrary.org/obo/CL_1001588) (I think this term should be obsoleted)
- [duodenum secretory cell](http://purl.obolibrary.org/obo/CL_1001589) (Cell definition is not good, but it doesn't looks restricted to glands)
- [epididymis glandular cell](http://purl.obolibrary.org/obo/CL_1001590)
- [gallbladder glandular cell](http://purl.obolibrary.org/obo/CL_1001592)
- [glandular cell of esophagus](http://purl.obolibrary.org/obo/CL_0002657) (this cell could be right, but looking at the daughter cells it doesn't look like)
- [glandular cell of the large intestine](http://purl.obolibrary.org/obo/CL_0002658) (it is not specific for glands)
- [glandular epithelial cell of stomach](http://purl.obolibrary.org/obo/CL_0002659)
- [oviduct glandular cell](http://purl.obolibrary.org/obo/CL_1001591)
- [parathyroid glandular cell](http://purl.obolibrary.org/obo/CL_1001593)
- [parathyroid glandular cell](http://purl.obolibrary.org/obo/CL_1001593)
- [rectum glandular cell](http://purl.obolibrary.org/obo/CL_1001595)
- [seminal vesicle glandular cell](http://purl.obolibrary.org/obo/CL_1001597)
- [uterine cervix glandular cell](http://purl.obolibrary.org/obo/CL_1001587)

Reply:
I've started to work through these.  So far parathyroid glandular cell.  I'm still convinced that limiting this to secretory cells is the conservative approach.  We have many epithelial cells of glands that have never been under this term, and lots of terms for secretory epithelial cells that are not part of glands were under this term before the refactor.

@Caroline-99  will work through the rest.

Reply:
- [oviduct glandular cell](http://purl.obolibrary.org/obo/CL_1001591) - the defs mentions peg cells and ciliated cells which are not restricted to glands. 
- [uterine cervix glandular cell](http://purl.obolibrary.org/obo/CL_1001587) - lines the endocervix (not a gland but cells are located in structures that function as glands (the endocervical crypts). https://www.ncbi.nlm.nih.gov/books/NBK568392/
- [rectum glandular cell](http://purl.obolibrary.org/obo/CL_1001595) - the definition includes examples that don't fit the new defs - ie. enterocytes/absorptive cells, M cells. Definition needs improving.
-  [parathyroid glandular cell](http://purl.obolibrary.org/obo/CL_1001593) - fits the new definition